cg/cli/workflow/balsamic/deliver.py

```diff
@@ -6,11 +6,15 @@
 
 import click
 from cg.apps import hk, lims
-from cg.meta.deliver.balsamic import DeliverAPI
+from cg.meta.deliver import DeliverAPI
 from cg.store import Store
 
 LOG = logging.getLogger(__name__)
 
+CASE_TAGS = []
+
+SAMPLE_TAGS = ["bam", "bam-index", "cram", "cram-index"]
```

Comment:
yes, for downsampled MT files and I guess backwards compatability with old bundles (for mip-dna). But I guess this will be updated for balsamic when it goes live. I think we can safely ignore it in this PR.